shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -1621,6 +1634,11 @@ - (NSString*)contrastMode {
   }
 }
 
+- (BOOL)captureTextFromCameraEnabled {
+  UITextField* textField = [self textField];
+  return [textField canPerformAction:@selector(captureTextFromCamera:) withSender:nil];
```

Comment:
Is there a notification you can listen to, if the value you get here changes?

Reply:
For instance will the value become false, if the user rejects the camera permission?

Reply:
This api invoke system internal camera API and it seems that is will not ask users for camera permission..

Reply:
[iOSDemo 2.zip](https://github.com/flutter/engine/files/8810283/iOSDemo.2.zip)
Here is demo code. I didn't request any permission

Reply:
Without the capability of monitoring the results of this `canPerformAction:` call I think it would be better to implement this feature in a way that's similar to paste: introduce an API that allows the developer to query whether live text is currently enabled on the platform, otherwise they may get outdated value?

Reply:
Got it. I will take a look. Thx

Reply:
Ah I thought you could still turn camera access off in Settings? (don't have a test device available atm, I'll give this a try tomorrow)

Reply:
Yeah if you disable camera access in Settings -> Screen Time -> Content & Privacy Restrictions -> Allowed Apps, the text edit menu item will be gone.